.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: java
```

Comment:
Out of curiosity, what is Travis doing for us here beyond running `html5validator` and why is Java the appropriate language setting for running that python script?